softgroup/basis/common-dao/common-dao-impl/src/main/java/com/softgroup/common/dao/impl/service/ContactService.java

```diff
@@ -4,11 +4,30 @@
 import com.softgroup.common.dao.impl.repositories.ContactRepository;
 import org.springframework.stereotype.Component;
 
+import java.util.ArrayList;
+import java.util.List;
+
 /**
  * Author: vadym
  * Date: 07.03.17
  * Time: 16:40
  */
 @Component
 public class ContactService extends BaseService<ContactEntity, ContactRepository> {
+
+    public ContactEntity findByNameAndProfileId(String name, String profileId) {
+        return new ContactEntity();
```

Comment:
@VadymPolyanski findByNameAndProfileId... New entity... Did you smoke something ?